CONTRIBUTING.md

```diff
@@ -78,7 +78,7 @@ rendering documentation on its website. The configuration file (`conf.py`) for
 `sphinx` can be found
 [under the `docs` folder](https://github.com/glass-dev/glass/blob/main/docs/conf.py).
 The documentation is deployed on <https://readthedocs.io>
-[here](https://glass.readthedocs.io/en/latest/).
+[here](https://glass.readthedocs.io/en/latest).
```

Comment:
Any reason for touching this line?

Reply:
I dislike the trailing slash, and was semi-relevant as a RTD link

Reply:
But the URL is `https://glass.readthedocs.io/en/latest/`, the version without slash is just a forward.